### ISSUE TITLE
[QUESTION] Why otp field and not the KeePass fields TimeOtp-Secret or TimeOtp-Secret-Base32

### ISSUE THREAD

**Issue by HalliHalloSchatz** [PhilippC/keepass2android#2330]
Newest Version installed 

Why otp field and not the KeePass fields TimeOtp-Secret or TimeOtp-Secret-Base32? 

When I used the KeePass fields, than KeePass show me options for OTP. With only otp field KeePass show me no OTP options. 

Keepass2Android: with KeePass field and no otp field OTP will shown. 

Why do not KeePass2Android create and used the KeePass fields?

**Comment by PhilippC** [comment]
this is now explained on https://github.com/PhilippC/keepass2android/blob/master/docs/Generating-TOTPs.md

**Comment by jsbiff** [comment]
I followed that link, which says that the official KeepPass totp field should work with Keepass2Android. I have already setup totp for one account in Windows Keepass2 (the official app), and it works in Windows, and I synchronized the database (via dropbox) to Keepass2Android, and now for that entry, a new field shows up, "TimeOtp-Secret-Base32", however, there is no field to see/copy the generated OTP? How to get this working so I can do OTP logins via either desktop or Android app?

When I go to the K2A Settings->About, it shows version "1.10-pre", and Play Store shows no updates. Is that an older version that does not yet support this functionality? Will you be pushing an update to Google Play soon?
